tests/python/driver/tvmc/test_tvmc_common.py

```diff
@@ -273,3 +273,24 @@ def test_parse_multiple_target_with_opts():
     assert "myopt" in targets[0]["opts"]
     assert "value" == targets[0]["opts"]["myopt"]
     assert "llvm" == targets[1]["name"]
+
+
+def test_parse_multiple_separators_on_target():
```

Comment:
IMHO, it's fine to put them to one test. You could put the current test name to be the error message when assertation failure of each case.

Reply:
I merged them into a single test, which I called `test_parse_quotes_and_separators_on_options`, but I think the error in case it fails, is more helpful if we just leave the default "expecting X but got Y instead", sort of message.

Example:
```
>       assert "+v1.0x,+value" == targets[0]["opts"]["option1"]
E       assert '+v1.0x,+value' == "'+v1.0x,+value'"
E         - '+v1.0x,+value'
E         ? -             -
E         + +v1.0x,+value
```

Reply:
@comaniac I'll adjust the test into the same PR with clarification of the regex :)